synapse/datamodel.py

```diff
@@ -518,7 +518,8 @@ def __init__(self):
         item = s_types.NodeProp(self, 'nodeprop', info, {})
         self.addBaseType(item)
 
-        info = {'doc': 'A potentially huge/tiny number. [x] <= 170141183460469231731687 with a fractional precision of 15 decimal digits.'}
+        info = {'doc': 'A potentially huge/tiny number. [x] <= 170141183460469231731687 with a fractional '
+                       + 'precision of 15 decimal digits.'}
```

Comment:
what is this extra plus sign???

Reply:
Do you prefer it without the plus?

Reply:
multiline strings should be automatically concatenated together?

Reply:
ok